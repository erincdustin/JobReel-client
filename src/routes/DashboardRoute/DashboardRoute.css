.dashboard-page {
  display: grid;
  grid-template-columns: 1fr 1fr 1fr 1fr 1fr 1fr 1fr 1fr;
  grid-gap: 1em;
}

.title {
  grid-column-start: 1;
  grid-column-end: 9;
  text-align: center;
}