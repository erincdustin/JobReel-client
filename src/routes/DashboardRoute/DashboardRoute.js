import React from 'react';
import './DashboardRoute.css';
import JobReelContext from '../../context/JobReelContext';
import SideNav from '../../components/SideNav/SideNav';
import SummaryContainer from '../../components/SummaryContainer/SummaryContainer';
import JobsList from '../../components/JobsList/JobsList'

class DashboardRoute extends React.Component {
  // static defaultProps = {
  //   location: {},
  //   history: {
  //     push: () => { },
  //   },
  // }

  static contextType = JobReelContext;

  render() {
    return(
      <div className='dashboard-page'>
        <div className='title'>
          <h2>Dashboard</h2>
        </div>
        <SideNav />
        <SummaryContainer />
        {/* <JobsList /> */}
      </div>
    )
  }
}

export default DashboardRoute