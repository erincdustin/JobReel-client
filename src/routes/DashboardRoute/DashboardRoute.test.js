import React from 'react'
import ReactDOM from 'react-dom'
import {BrowserRouter} from 'react-router-dom'
import DashboardRoute from './DashboardRoute'
import { shallow } from 'enzyme';
import toJson from 'enzyme-to-json';

describe('DashboardRoute smoke test', () => {
  it('renders without crashing', () => {
    const div = document.createElement('div')
    ReactDOM.render(<BrowserRouter><DashboardRoute /></BrowserRouter>, div)
    ReactDOM.unmountComponentAtNode(div)
  })
})

describe('DashboardRoute snapshot test', () => {
  it('renders the UI as expected', () => {
    const wrapper = shallow(<DashboardRoute />)
    expect(toJson(wrapper)).toMatchSnapshot();
  });
})