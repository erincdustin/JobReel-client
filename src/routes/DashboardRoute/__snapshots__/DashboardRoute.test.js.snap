// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`DashboardRoute snapshot test renders the UI as expected 1`] = `
<div
  className="dashboard-page"
>
  <div
    className="title"
  >
    <h2>
      Dashboard
    </h2>
  </div>
  <div
    className="dashboard-image"
  >
    <img
      alt="summary-background"
      src="summary.svg"
    />
  </div>
  <SideNav />
  <SummaryContainer />
</div>
`;
